pkg/spec/cluster.go

```diff
@@ -142,8 +144,9 @@ type PodPolicy struct {
 // AuthConfig is the authorization configuration for
 // user permissions in the cluster.
 type AuthConfig struct {
-	ClientsAuthSecret  string `json:"clientsAuthSecret,omitempty"`
-	ClientsAuthTimeout int    `json:"clientsAuthTimeout,omitempty"`
+	EnableServiceAccounts bool   `json:"enableServiceAccounts,omitempty"`
```

Comment:
It may be good to add a few lines of doc about `EnableServiceAccounts` and `ClientsAuth` modes of auth and what they do.